### DISCUSSION TITLE
[求助]我的2手14pro被锁有绳匠帮助吗

### DISCUSSION THREAD

**Discussion by daofeng9612** [share121/inter-knot#3233] [其他]
骗子要求2000丁尼给解开求绳匠帮忙谢谢
![idlefish-msg-1723705769406 png](https://github.com/user-attachments/assets/6cf3400c-e20d-40a9-b5b0-44b04f88c35a)

**Comment by dch-cm** [comment]
新艾利都治安局提醒您，为保障市民权益，请于正规平台交易

**Reply by daofeng9612** [reply]
谢谢

**Comment by saiki816** [comment]
爱思刷机，激活的时候用锁屏密码，网上都有教程的，但你是怎么被锁的，能讲讲吗

**Reply by daofeng9612** [reply]
这个办法不行当时收到有个要购买灵动岛截图的之后就被锁了

**Reply by daofeng9612** [reply]
锁完之后我想抹除数据就开了就恢复出厂了

**Reply by saiki816** [reply]
那你手机现在可以正常用了？

**Reply by daofeng9612** [reply]
不

**Reply by daofeng9612** [reply]
> 那你手机现在可以正常用了？

我刚发的帖子

**Reply by saiki816** [reply]
那现在还是你拍的那张图？

**Reply by saiki816** [reply]
还是说卡在激活页面进不去

**Reply by daofeng9612** [reply]
卡在激活了

**Reply by saiki816** [reply]
那点激活后应该会有个输入appleid的页面，在那里输入骗子的appleid和密码，如果不行就点中间蓝色字的“使用设备密码解锁”，如果没有那就丸啦！

**Reply by daofeng9612** [reply]
我还真没屏幕密码

**Reply by daofeng9612** [reply]
> 那点激活后应该会有个输入appleid的页面，在那里输入骗子的appleid和密码，如果不行就点中间蓝色字的“使用设备密码解锁”，如果没有那就丸啦！

🥲

**Reply by saiki816** [reply]
> 我还真没屏幕密码

那丸啦！只能当废品卖了，或者去试试apple客服？

**Reply by saiki816** [reply]
说不定他们有后台可以帮你

**Reply by daofeng9612** [reply]
2手官方真的可以解吗

**Reply by saiki816** [reply]
> 2手官方真的可以解吗

不清楚，但还是去试试吧，看看能不能抓住这最后的稻草

**Reply by daofeng9612** [reply]
好的谢谢麻烦绳匠了

**Reply by saiki816** [reply]
以后注意，别随随便便的就去登别人的appleid了，还有那个钱千万别转，可能还是个无底洞

**Reply by daofeng9612** [reply]
谢谢绳匠提醒

**Reply by saiki816** [reply]
如果那个客服都不行的话那就直接去报警吧，毕竟是四五千

**Reply by daofeng9612** [reply]
那个骗子也没骗到钱我们这不能立案

**Reply by saiki816** [reply]
你试过了？

**Reply by daofeng9612** [reply]
嗯

**Reply by saiki816** [reply]
那就去试试手机店，说不定有科技能帮你解锁

**Reply by daofeng9612** [reply]
好的绳匠

**Reply by Hongxuli2006** [reply]
楼主，如果你看到的话，请回复我一下
如果你没解决问题，希望能对你有帮助
第一，本人经常在朋友向我借取美国iOS ID的时候多次强调，永远不要登录设置！！！不然的话apple ID号主可以利用“查找我的iPhone”功能封锁你的apple 并且借此机会狠狠敲榨。

第二，如果想要解锁的话，爱思助手可能是一种方法，但是可能并不是最佳方案，这点取决于你苹果型号。但是最确定的是，苹果商店是非常好的解决方案。不过请一定找到当初的收据，装苹果的盒子。这两个缺一不可，缺少一个可能苹果工作人员会以“无法证明手机机主身份”的理由来拒绝协助你！

希望能够帮到你
如果有疑问可以来联系我
我的邮箱是hongxuli401@gmail.com

**Comment by SakuraLainMaid** [comment]
提醒一下，只要你有办法证明手机属于你的（发票，购买凭证，手机包装盒 ，etc）你可以在世界范围内任何一个genius bar解锁

**Comment by ykrsama** [comment]
这个委托我接了！
